hack/generator/pkg/astmodel/importer.go

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package astmodel
+
+type PackageImporter interface {
```

Comment:
I don’t much like this as an interface because we never abstract over it (`PackageImporter` is never used as an argument/return type)

Reply:
You prefer just to define 2x the same method (one on each?)